test/test_vmap.py

```diff
@@ -3530,6 +3529,7 @@ def test_distributions(self, device, randomness, use_generator, batched_input):
         supported_ops = [
             lambda _, t: torch.normal(t, 1., **kwargs),
             lambda _, t: torch.normal(0., torch.abs(t), **kwargs),
+            lambda _, t: torch.normal(t, torch.abs(t), **kwargs),
```

Comment:
torch.normal(a, b) 

For Tensors a and b, there are 12 cases to test:
- {error, same, different} x {a batched, b batched, a and b batched, nothing is batched}

I think we're only testing {error, same, different} x {a and b batched}?